pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotInstanceRestletResource.java

```diff
@@ -390,4 +391,33 @@ public SuccessResponse updateBrokerResource(
           Response.Status.INTERNAL_SERVER_ERROR, e);
     }
   }
+
+  @POST
+  @Path("/instances/safetyCheck/dropInstance")
```

Comment:
`/instances/dropInstance/validate` maybe a better path?

We do have `POST /schemas/validate` & `POST /tables/validate`

Reply:
yeah that's much better. Updated the path.